.gitignore

```diff
@@ -10,4 +10,7 @@ dist
 #Hardhat files
 cache
 artifacts
-typechain-types
\ No newline at end of file
+typechain-types
+
+# Ignore all .tgz files
+*.tgz
```

Comment:
Seems github didn't pickup that the master branch is updated and still shows all these changes that are already merged? Not sure why - @MrishoLukamba can you refresh it or close the PR and create a new one? Also you know you can create branches in the main repo and don't have to PR cross forks?